rsconcept/backend/shared/messages.py

```
@@ -42,6 +42,30 @@ def duplicateSchemasInArguments() -> str:
+
+
+def operationHasDependents(alias: str) -> str:
+    return f'Операция используется другими операциями: {alias}'
+
+
+def schemaReferencedByOperations() -> str:
+    return 'Схема используется другими операциями'
+
+
+def schemaNotInOSS() -> str:
+    return 'Схема не является результатом операции в этой ОСС'
```

Ruff (0.16.0)

[warning] String contains ambiguous `О` (CYRILLIC CAPITAL LETTER O). Did you mean `O` (LATIN CAPITAL LETTER O)?

(RUF001)